core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -537,27 +531,15 @@ trait WhiskWebActionsApi extends Directives with ValidateRequestSize with PostAc
    */
   private def verifyWebAction(actionName: FullyQualifiedEntityName, authenticated: Boolean)(
     implicit transid: TransactionId) = {
-    for {
-      // lookup the identity for the action namespace
-      actionOwnerIdentity <- identityLookup(actionName.path.root) flatMap { i =>
-        entitlementProvider.checkThrottles(i) map (_ => i)
-      }
 
-      // lookup the action - since actions are stored relative to package name
-      // the lookup will fail if the package name for the action refers to a binding instead
-      // also merge package and action parameters at the same time
-      // precedence order for parameters:
-      // package.params -> action.params -> query.params -> request.entity (body) -> augment arguments (namespace, path)
-      action <- confirmExportedAction(actionLookup(actionName), authenticated) flatMap { a =>
-        if (a.namespace.defaultPackage) {
-          Future.successful(a)
-        } else {
-          pkgLookup(a.namespace.toFullyQualifiedEntityName) map { pkg =>
-            (a.inherit(pkg.parameters))
-          }
+    // lookup the identity for the action namespace
+    identityLookup(actionName.path.root) flatMap { actionOwnerIdentity =>
```

Comment:
here the identity is tied to how the action is invoked: if the path is relative to the package, it's the owner of the package. But if it's via the binding, the identity is the owner binding. The two identities are not the same. It might also not make sense to invoke an action in a public package since it might require a binding to work correctly.

Reply:
another aspect to document: a web action in a public package cannot be disabled so any (private) binding of the package will have the web action exposed.